.github/pull_request_template.md

```diff
@@ -13,21 +13,14 @@ the terms of the Apache 2.0 license.
 
 ## PR Checklist
 
-`[Author TODO: Meet these criteria. Where there are two options, keep one.]`
+`[Author TODO: Meet these criteria.]`
 `[Reviewer TODO: Verify that these criteria are met. Request changes if not]`
 
+- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).
```

Comment:
This looks out of place here, it's not really a criteria the PR should meet.

I'd add it in its own category.

Reply:
Moved it to the `## Description of Changes` category - maybe while the contributor is writing the description, they can be prompted this way to consider sharing on the broader context.